misc/deployments/test4.gno.land/config.toml

```
@@ -0,0 +1,247 @@
+  cors_allowed_methods = ["HEAD", "GET", "POST", "OPTIONS"]
+
+  # A list of origins a cross-domain request can be executed from
+  # Default value '[]' disables cors support
+  # Use '["*"]' to allow any origin
+  cors_allowed_origins = ["*"]
+
+  # TCP or UNIX socket address for the gRPC server to listen on
+  # NOTE: This server only supports /broadcast_tx_commit
+  grpc_laddr = ""
+
+  # Maximum number of simultaneous connections.
+  # Does not include RPC (HTTP&WebSocket) connections. See max_open_connections
+  # If you want to accept a larger number than the default, make sure
+  # you increase your OS limits.
+  # 0 - unlimited.
+  # Should be < {ulimit -Sn} - {MaxNumInboundPeers} - {MaxNumOutboundPeers} - {N of wal, db and other open files}
+  # 1024 - 40 - 10 - 50 = 924 = ~900
+  grpc_max_open_connections = 900
+  home = ""
+
+  # TCP or UNIX socket address for the RPC server to listen on
+  laddr = "tcp://127.0.0.1:26657" # Change me!
+
+  # Maximum size of request body, in bytes
+  max_body_bytes = 1000000
+
+  # Maximum size of request header, in bytes
+  max_header_bytes = 1048576
+
+  # Maximum number of simultaneous connections (including WebSocket).
+  # Does not include gRPC connections. See grpc_max_open_connections
+  # If you want to accept a larger number than the default, make sure
+  # you increase your OS limits.
+  # 0 - unlimited.
+  # Should be < {ulimit -Sn} - {MaxNumInboundPeers} - {MaxNumOutboundPeers} - {N of wal, db and other open files}
+  # 1024 - 40 - 10 - 50 = 924 = ~900
+  max_open_connections = 900
+
+  # How long to wait for a tx to be committed during /broadcast_tx_commit.
+  # WARNING: Using a value larger than 10s will result in increasing the
+  # global HTTP write timeout, which applies to all connections and endpoints.
+  # See https://github.com/tendermint/classic/issues/3435
+  timeout_broadcast_tx_commit = "10s"
+
+  # The path to a file containing certificate that is used to create the HTTPS server.
+  # Might be either absolute path or path related to tendermint's config directory.
+  # If the certificate is signed by a certificate authority,
+  # the certFile should be the concatenation of the server's certificate, any intermediates,
+  # and the CA's certificate.
+  # NOTE: both tls_cert_file and tls_key_file must be present for Tendermint to create HTTPS server. Otherwise, HTTP server is run.
+  tls_cert_file = ""
+
+  # The path to a file containing matching private key that is used to create the HTTPS server.
+  # Might be either absolute path or path related to tendermint's config directory.
+  # NOTE: both tls_cert_file and tls_key_file must be present for Tendermint to create HTTPS server. Otherwise, HTTP server is run.
+  tls_key_file = ""
+
+  # Activate unsafe RPC commands like /dial_seeds and /unsafe_flush_mempool
+  unsafe = false
+
+##### node telemetry #####
+[telemetry]
+  enabled = false # Advised to be `true`
```

```suggestion
  enabled = true # Advised to be `true`
```

Let's ask early testers to set up telemetry as well. This is a good opportunity for power users to learn more about observing a running node.

---

Enabled by default:

[42781fc](https://github.com/gnolang/gno/pull/2508/commits/42781fc0dd7a55dd6484bc071499b7a18b3ffc59)